dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7806,6 +7806,56 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey DORA_WORKER_METASTORE_ROCKSDB_BLOOM_FILTER =
+      booleanBuilder(Name.DORA_WORKER_METASTORE_ROCKSDB_BLOOM_FILTER)
+          .setDescription("Whether or not to use a bloom filter in the Block meta"
+              + " table in RocksDB. If unset, the RocksDB default will be used."
+              + " See https://github.com/facebook/rocksdb/wiki/RocksDB-Bloom-Filter")
+          .setDefaultValue(false)
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
+          .setScope(Scope.WORKER)
+          .build();
+  public static final PropertyKey DORA_WORKER_METASTORE_ROCKSDB_CACHE_SIZE =
+      longBuilder(Name.DORA_WORKER_METASTORE_ROCKSDB_CACHE_SIZE)
+          .setDescription("The capacity in bytes of the RocksDB block metadata table LRU "
+              + " cache. If unset, the RocksDB default will be used."
+              + " See https://github.com/facebook/rocksdb/wiki/Block-Cache."
+              + " Our microbench test shows that 1GB cache can double the metadata read operation"
+              + " performance for a meta store that persists 10M metadata entries."
+              + " Note that the memory is off-heap and does not count in jvm memory.")
+          .setDefaultSupplier(
```

Comment:
how to set up the default value?
this is the offheap memory so assigning 1/10 of the heap memory might also cause OOM. We can probably default a conservative one (like 128mb) for this by default as the profiling results show that even a rocks db with 32mb cache can satisfy our requirements 

@beinan thoughts?